composer/yamls/models/glue/stsb.yaml

```diff
@@ -52,3 +52,4 @@ callbacks:
 load_path: https://storage.googleapis.com/llm_checkpoints/bert_checkpoint/bert_checkpoints/ep7.pt
 load_weights_only: true
 load_strict_model_weights: false
+precision: fp32 # Eval requires FP32
```

Comment:
Is this only required for STS-B? Or is this an impact of the precision refactor and we should avoid this long-term?

Reply:
It seems to be a consequence of the precision refactor (STS-B wasn't causing problems before that), but the issue remains after merging in dev after a fix was pushed. It's unclear to me what the root of the issue is, but it has something to do with the `SpearmanCorrCoeff` metric wanting the model predictions to be `fp32`. I'm not sure how to prevent this issue at the code level, but using `fp32` prevents the error and adds only a small overhead for this quick fine-tuning task.

Given the minor impact, I was hoping we could leave this as a TODO. If it needs to be dealt with now, I'm open to suggestions on how to proceed :)

Reply:
What concerns me isn't the small change on this PR, but rather, it indicates that something about the root cause of the issue still isn't fixed, and it'll be way worse if an open-source user is running into this -- it's highly less likely that they won't know what to do! In other words, we can continue being productive but Composer overall will suffer.

Since we can run our weekend runs without this, @mvpatel2000 thoughts on working with Alex on root-causing and finding an appropriate Composer solution that doesn't require `precision: fp32` here?

Reply:
Very good points!